packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx

```diff
@@ -84,6 +98,27 @@ export const NodeTable = React.memo((props: Props) => {
 
   const rowData = nodes.map((thisNode) => nodeToRow(thisNode, linodes ?? []));
 
+  // It takes ~5 minutes for LKE-E cluster nodes to be provisioned and we want to explain this to the user
+  // since nodes are not returned right away unlike standard LKE
+  const lkeEClusterCreatedWithinFirst10Mins = () => {
```

Comment:
```suggestion
  const isEnterpriseClusterWithin10MinsOfCreation = () => {
```

What do you think about this function name - more or less confusing to read? The `is` makes it a little more explicit about what the return value is. I had a little trouble parsing the current function name when I first read it, and I think that + the "CreatedWithinFirst" were what tripped me up.

Reply:
Makes sense to me!